src/use-swr-pages.tsx

```diff
@@ -1,13 +1,12 @@
-import React, { useCallback, useMemo, useState, useRef } from 'react'
-
-import { cacheGet, cacheSet } from './config'
+import React, { useEffect, useCallback, useMemo, useState, useRef } from 'react'
 import {
   pagesResponseInterface,
   responseInterface,
   pageComponentType,
   pageOffsetMapperType
 } from './types'
 
+import { cacheGet, cacheSet } from './config'
```

Comment:
This should be
```suggestion
import { cache } from './config'
```
Then use cache.get and cache.set instead

Reply:
thanks, hadn't pulled the latest changes, will integrate